python/lsst/ts/ofc/bending_mode.py

```diff
@@ -19,41 +19,47 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
-__all__ = ["BendModeToForce"]
+__all__ = ["BendingMode"]
 
 import numpy as np
 
 from .ofc_data import OFCData
 from .utils import rot_1d_array
 
 
-class BendModeToForce:
-    """Transform Bend mode to forces.
+class BendingMode:
```

Comment:
Why the name change?

Reply:
Because BendingModeToForce is not exactly the case anymore. Now we can also go from bendingMode to stresses, so that's why it made more sense conceptually to change the name.